prime-dotnet-webapi/Services/EnrolleeAgreementService.cs

```diff
@@ -17,16 +20,19 @@ public class EnrolleeAgreementService : BaseService, IEnrolleeAgreementService
 
         private readonly IAgreementService _agreementService;
         private readonly IRazorConverterService _razorConverterService;
+        private readonly IMapper _mapper;
 
         public EnrolleeAgreementService(
             ApiDbContext context,
             ILogger<EnrolleeAgreementService> logger,
             IAgreementService agreementService,
-            IRazorConverterService razorConverterService)
+            IRazorConverterService razorConverterService,
+            IMapper mapper)
```

Comment:
I think we're doing those in alphabetical order (except for context and logger they're always on top).